src/App.tsx

```diff
@@ -1,156 +1,87 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState, useMemo } from 'react';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { getTodos } from './api/todos';
+import { Header } from './components/Header/Header';
+import { TodoList } from './components/TodoList/TodoList';
+import { Footer } from './components/Footer/Footer';
+import { FilterStatus } from './types/FilterStatus';
+import { ErrorBlock } from './components/ErrorBlock/ErrorBlock';
 
-const USER_ID = 0;
+const USER_ID = 10266;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [error, setError] = useState<string | null>(null);
```

Comment:
as initial value you can use empty string and it also gives false in conditions
```suggestion
  const [error, setError] = useState<string>('');
```